scripts/prepare-release.js

```diff
@@ -332,6 +333,57 @@ async function updateBackstageReleaseVersion(repo, type) {
   );
 }
 
+/**
+ * Ensures that the changesets include a version bump of create-app otherwise
+ * generates a new patch changeset for create-app.
+ */
+async function ensureCreateAppChangeset() {
+  const changesetPath = path.resolve(__dirname, '../.changeset');
+  const fileNames = await fs.readdir(changesetPath);
+  const changesetNames = fileNames.filter(
+    name => name.endsWith('.md') && name !== 'README.md',
+  );
+
+  const changesets = await Promise.all(
+    changesetNames.map(async name => {
+      const content = await fs.readFile(path.join(changesetPath, name), 'utf8');
+      return { name, ...parseChangeset(content) };
+    }),
+  );
+
+  let excludeList = [];
+  const prePath = path.resolve(changesetPath, 'pre.json');
+  if (await fs.pathExists(prePath)) {
+    const data = await fs.readJSON(prePath);
+    // Only exclude changesets in pre-release mode.
+    excludeList =
+      data.mode === 'pre' ? data.changesets.map(name => `${name}.md`) : [];
```

Comment:
reassigns the empty array, could have done an `if` + `excludeList.push(...data.changesets.map(?))` instead

Reply:
Yes, good points! Fixed both 👍